### PR TITLE
Add definitions for factory builder terms

### PR DESCRIPTION
**Pull request by TimJentzsch**
Towards #151.

This PR adds glossary definitions for factory builder terms.

WIP.

### CODE REVIEW

File: design/src/glossary.md [hunk 1]
```diff
@@ -39,12 +39,55 @@ Terms defined here are standard elements of the factory builder genre.
 
 ### Belt
```
Comment (alice-i-cecile) [inline design/src/glossary.md:40]: Belts can have sides, which is an important gameplay difference wihin the genre.

File: design/src/glossary.md [hunk 2]
```diff
@@ -39,12 +39,55 @@ Terms defined here are standard elements of the factory builder genre.
...
+
+**Examples**:
+
+- [Factorio: Blueprint](https://wiki.factorio.com/Blueprint)
+- [Factorio: Ghost](https://wiki.factorio.com/Ghost)
+- [Satisfactory: Blueprint designer](https://satisfactory.fandom.com/wiki/Blueprint_Designer)
+
 ### Clipboard
 
+Similar to blueprints, the clipboards allows the quick reuse of factory layouts.
```
Comment (alice-i-cecile) [inline design/src/glossary.md:79]:
```suggestion
Similar to blueprints, the clipboard allows the quick reuse of factory layouts.
```

**Comment by alice-i-cecile** [comment]
Excellent start, thank you.

**Review by bzm3r — COMMENTED** [review]
Great start!

**Comment by bzm3r** [comment]
Why is this draft? Will you be adding more terms? If you want, you can undraft it, and I am sure it will be reviewed/merged super quick.

**Comment by alice-i-cecile** [comment]
Yeah I'm fine to either merge this aggresively or wait a bit longer for you to finish filling out this batch of terms. Definitely don't want this hanging around for more than a couple days though :)

**Comment by TimJentzsch** [comment]
I wanted to finish the rest of the factory terms before merging, but I had to leave :)
I'll finish this today or tomorrow.

Mostly opened this as draft so that we don't accidentally duplicate work.

File: design/src/glossary.md (hunk 1 above)
Comment (TimJentzsch) [reply design/src/glossary.md:40]: Do you mean the two-side split in Factorio or the direction the belt is facing?

**Review by wallamide — COMMENTED** [review]
Glad to see this glossary getting added! This looks good so far. Thanks for working on the draft!

**Comment by TimJentzsch** [comment]
I can't think of good definitions for "Overlay", "Resource" and "Logistic network" right now, if you have some feel free to add them :)

Otherwise this is ready for review now.

File: design/src/glossary.md (hunk 1 above)
Comment (alice-i-cecile) [reply design/src/glossary.md:42]: The two-side split.

**Review by bzm3r — APPROVED** [review]
😍

File: design/src/glossary.md [hunk 3]
```diff
@@ -39,36 +41,173 @@ Terms defined here are standard elements of the factory builder genre.
...
+Some actions can only be automated through bots, e.g. constructing new entities.
+
+**Examples**:
+
+- [Factorio: Construction robot](https://wiki.factorio.com/Construction_robot)
+- [Factorio: Logistic robot](https://wiki.factorio.com/Logistic_robot)
+
 ### Blueprint
 
+Blueprints are placeholders for multiple entities to be constructed.
```
Comment (alice-i-cecile) [inline design/src/glossary.md:69]: This should also mention that entity settings are saved.

File: design/src/glossary.md [hunk 4]
```diff
@@ -39,36 +41,173 @@ Terms defined here are standard elements of the factory builder genre.
...
-### Train
+An item that can be processed further to produce different items.
 
-### Research
+There are different types of resources:
+
+- **Raw resource**: A foundational resource that can be extracted directly from the environment.
+- **Intermediate resource**: A resource that cannot be extracted nor used directly.
+  It's an intermediate set towards the production of end products.
+- **End product**: The final stage of a resource, can be used directly.
```
Comment (alice-i-cecile) [inline design/src/glossary.md:186]:
```suggestion
- **End product**: A resource that can be used directly for some in-game benefit.
```

File: design/src/glossary.md [hunk 5]
```diff
@@ -39,36 +41,173 @@ Terms defined here are standard elements of the factory builder genre.
...
+
+**Examples**:
+
+- [Factorio: Science pack](https://wiki.factorio.com/Science_pack)
+- [Dyson Sphere Program: Science matrices](https://dyson-sphere-program.fandom.com/wiki/Category:Science_Matrices)
+
+### Train
+
+An entity to efficiently move big amounts of resources over large distances.
+Usually require a much higher upfront investment than _belts_, but are better suited for long distances.
```
Comment (alice-i-cecile) [inline design/src/glossary.md:203]:
```suggestion
Usually require a much higher upfront investment than _belts_, but are better suited for long distances.
They are high latency, and move goods in clumped bursts.
```

**Pull request merged**